src/tooling/piral-cli-parcel/README.md

```diff
@@ -1,6 +1,4 @@
-[![Piral Logo](https://github.com/smapiot/piral/raw/master/docs/assets/logo.png)](https://piral.io)
```

Comment:
Why this change?

All our READMEs look similar. Compare:

https://raw.githubusercontent.com/smapiot/piral/develop/src/tooling/piral-cli/README.md
(same for `piral-cli-webpack`)

If the change was made due to the logo etc. being rendered (and looking ugly), please look at how plugins are rendered / done. You can customize this.

Reply:
rolled back these 2 changes